src/screens/Event/EventAdd.js

```diff
@@ -286,6 +286,20 @@ type Props = {
     this.timer = null;
   }
 
+  componentDidMount() {
+    const isEdit = this.props.query.event;
+
+    if (isEdit) {
+      const { publicLimit } = this.props.query.event;
+
+      if (publicLimit) {
```

Comment:
avoid nested `if`

Reply:
I Agree with @jgcmarins this is not one of the best practice and can you explain this isEdit field is this at the graphql

Reply:
@steniowagner

Reply:
Ok for the "avoid nested if".

The reason for the 'isEdit' is that the user would edit the event, and, this flag comes from the server as an object, with all information about the event.

For me, this is not a good practice, we could use a single component to handle all the event operations: create, detail and edit, instead of use two different screens to handle the operations. (I did this, but my PR was denied).